postreise/plot/plot_upgrades_map.py

```diff
@@ -81,7 +82,7 @@ def _map_upgrades(
         {
             "xs": branch_all[["from_x", "to_x"]].values.tolist(),
             "ys": branch_all[["from_y", "to_y"]].values.tolist(),
-            "cap": branch_all["rateA"] * all_branch_scale + all_branch_min,
+            "cap": branch_all["rateA"] / 1000 * all_branch_scale + all_branch_min,
```

Comment:
Consider making the 1000 a module constant

Reply:
I'm trying to come up with a reasonable name, but they all seem a little silly, e.g. `num_MW_in_GW`. What about adding a new column `branch_all["rateA_GW"] = branch_all["rateA"] / 1000  # MW to GW` and then using that column later in the function?

Reply:
The new column would make sense but we use the same factor in several places, so was hoping to reuse that. What about calling it `scale_factor` and just putting a comment next to it?

Reply:
`scale_factor` seems pretty generic. What if we create new variables `all_branch_scale_MW` and `diff_branch_scale_MW`, calculate them at the top with the `/ 1000` logic and a comment, and then apply them everywhere?

Reply:
Sounds good to me

Reply:
Added in the latest commit.
https://github.com/Breakthrough-Energy/PostREISE/pull/271/commits/b1b7bc1d10dbf78d989f96dd76b5cb9557e9524f

Reply:
Thanks for indulging